sormas-backend/src/test/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjbPseudonymizationTest.java

```diff
@@ -185,8 +188,8 @@ private void assertNotPseudonymized(EventParticipantDto eventParticipant) {
 
 	private void assertPseudonymized(EventParticipantDto eventParticipant) {
 		assertThat(eventParticipant.getInvolvementDescription(), isEmptyString());
-		assertThat(eventParticipant.getPerson().getFirstName(), isEmptyString());
-		assertThat(eventParticipant.getPerson().getLastName(), isEmptyString());
+		assertEquals(I18nProperties.getCaption(Captions.inaccessibleValue), eventParticipant.getPerson().getFirstName());
```

Comment:
please use assertThat to don't break the logic

Reply:
updated